herbert_core/applications/multifit/@mfclass/private/multifit_lsqr_ser.m

```diff
@@ -0,0 +1,730 @@
+function [p_best,sig,cor,chisqr_red,converged]=multifit_lsqr_ser(w,xye,func,bfunc,pin,bpin,...
+                                                      f_pass_caller_info,bf_pass_caller_info,pfin,p_info,listing,fcp,perform_fit)
+    % Perform least-squares minimisation
+    %
+    %   >> [p_best,sig,cor,chisqr_red,converged]=...
```

Comment:
does not give correct function name

presuming _ser means serial but best to say it in the comment above